starfyre/compiler.py

```diff
@@ -2,13 +2,24 @@
 from pathlib import Path
 
 
-def get_fyre_files(project_dir):
+def get_fyre_files(project_dir):   
     fyre_files = []
     for file in os.listdir(project_dir):
         if file.endswith(".fyre"):
             fyre_files.append(file)
     return fyre_files
 
+def import_css(import_statement):
```

Comment:
Also, one suggestion we can name it as `def resolve_css_import(import_statement):`

Reply:
Thank you. Suggestion applied. :)